src/Core/Framework/Api/Controller/InfoController.php

```diff
@@ -58,7 +58,7 @@ public function __construct(
         private readonly SystemConfigService $systemConfigService,
         private readonly ApiRouteInfoResolver $apiRouteInfoResolver,
         private readonly InAppPurchase $inAppPurchase,
-        private readonly ViteFileAccessorDecorator $viteFileAccessorDecorator,
+        private readonly ?ViteFileAccessorDecorator $viteFileAccessorDecorator,
```

Comment:
I know, this is the simple solution. but how about decoupling it completely with an event or so?

Reply:
i won't go in that direction, actually the controller is only there fore the admin, or at least that route, so doing it "correctly" the route should be in admin namespace, but we don't want more PHP in admin I guess